compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/TreeModifier.java

```diff
@@ -725,14 +725,28 @@ public RequiredParameterNode transform(
             RequiredParameterNode requiredParameterNode) {
         NodeList<AnnotationNode> annotations =
                 modifyNodeList(requiredParameterNode.annotations());
-        Token asteriskToken =
-                modifyToken(requiredParameterNode.asteriskToken().orElse(null));
         Node typeName =
                 modifyNode(requiredParameterNode.typeName());
         Token paramName =
                 modifyToken(requiredParameterNode.paramName().orElse(null));
         return requiredParameterNode.modify(
-                requiredParameterNode.kind(),
+                annotations,
+                typeName,
+                paramName);
+    }
+
+    @Override
+    public IncludedRecordParameterNode transform(
+            IncludedRecordParameterNode includedRecordParameterNode) {
+        NodeList<AnnotationNode> annotations =
+                modifyNodeList(includedRecordParameterNode.annotations());
+        Token asteriskToken =
+                modifyToken(includedRecordParameterNode.asteriskToken());
+        Node typeName =
+                modifyNode(includedRecordParameterNode.typeName());
+        Token paramName =
+                modifyToken(includedRecordParameterNode.paramName().orElse(null));
+        return includedRecordParameterNode.modify(
```

Comment:
Looks like we can improve the formatting?

Reply:
This is an auto-generated code segment and this formatting issue is with other places also. (In this file)